dblookupext/miniblocksHandler_test.go

```diff
@@ -792,3 +792,100 @@ func TestMiniblocksHandler_getMiniblockMetadataByTxHashAndMbHash(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, []*MiniblockMetadata{partiallyExecutedMiniblock1, partiallyExecutedMiniblock2, partiallyExecutedMiniblock3}, miniblocksMetadata)
 }
+
+func TestMiniblocksHandler_updateMiniblockMetadataOnBlock(t *testing.T) {
+	t.Parallel()
+
+	completeMiniblock, completeMiniblockHash, completeMiniblockHeader := generateTestCompletedMiniblock()
+	partialMiniblock, partialMiniblockHash, partialMiniblockHeaders := generateTestPartialMiniblock()
+
+	mbHandler := createMockMiniblocksHandler()
+
+	header1 := generateHeader(37, 22933, *completeMiniblockHeader, *partialMiniblockHeaders[0])
+	headerHash1, err := core.CalculateHash(testMarshaller, testHasher, header1)
+	assert.Nil(t, err)
+
+	header2 := generateHeader(37, 22934, *partialMiniblockHeaders[1])
+	headerHash2, err := core.CalculateHash(testMarshaller, testHasher, header2)
+	assert.Nil(t, err)
+
+	err = mbHandler.commitMiniblock(header1, headerHash1, completeMiniblock)
+	assert.Nil(t, err)
+
+	err = mbHandler.commitMiniblock(header1, headerHash1, partialMiniblock)
+	assert.Nil(t, err)
+
+	t.Run("completed hash with invalid header hash should not call update", func(t *testing.T) {
+		err = mbHandler.updateMiniblockMetadataOnBlock(completeMiniblockHash, []byte("missing header hash"), func(mbMetadataOnBlock *MiniblockMetadataOnBlock) {
+			assert.Fail(t, "should have not called update handler")
+		})
+		assert.Equal(t, ErrNotFoundInStorage, err)
+	})
+	t.Run("completed hash with valid header hash should call update", func(t *testing.T) {
```

Comment:
`completed miniblock hash` instead of `completed hash` (and above).

Reply:
done